Makefile

```diff
@@ -324,4 +208,4 @@ FORCE:
 .ml.cmo:
```

Comment:
Do we still need the suffix rules?

Reply:
Probably not.

I was thinking that we could absorb most of the makefile into the prebuild program anyway. That should be more fun to maintain. We would then just use the makefile targets to pass information to prebuild.